schema_based_on_diagram.sql

```diff
@@ -0,0 +1,61 @@
+CREATE TABLE patients (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255),
+    date_of_birth DATE
+);
+
+CREATE TABLE treatments (
+    id SERIAL,
+    type VARCHAR(50),
+    name VARCHAR(50),
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE invoice_items (
+    id SERIAL PRIMARY KEY,
+    unit_price decimal,
+    quantity integer,
+    total_price decimal,
+    invoice_id integer,
+    treatment_id integer
+);
+
+CREATE TABLE invoice (
+    id SERIAL PRIMARY KEY,
```

Comment:
- [ ] Kindly change this table name to its plural form `invoices`. This is an active record convention of naming tables.